internal/model/repository_mock.go

```diff
@@ -40,8 +40,8 @@ func (mock *RepositoryMock) AddPostMortemUrl(ctx context.Context, channelName st
 	return args.Error(0)
 }
 
-func (mock *RepositoryMock) CancelIncident(ctx context.Context, channelID string, description string) error {
-	args := mock.Called(ctx, channelID, description)
+func (mock *RepositoryMock) CancelIncident(ctx context.Context, inc *Incident) error {
+	args := mock.Called(ctx, inc.ChannelId, inc.DescriptionCancelled)
```

Comment:
There was a reason for using this fields in the Called function? I think it could be
```suggestion
	args := mock.Called(ctx, inc)
```

Reply:
> There was a reason for using this fields in the Called function? I think it could be

I confess that I have not found the reason why it is passed for mock.Called. I just fixed the parameters that were passed to a function CancelIncident with a new signature